BetaCameras/Kinect4Azure/Kinect4Azure.cs

```diff
@@ -11,6 +10,10 @@
 using MetriCam2.Exceptions;
 using Microsoft.Azure.Kinect.Sensor;
 using System.Buffers;
+#if NETSTANDARD2_0
```

Comment:
```suggestion
#if !NETSTANDARD2_0
```
(and remove the `#else` line below)
(L195 uses `!`, too)

Reply:
Is it only possible to modify one line when providing suggestions?